### PR TITLE
add potential org verify/ignore UI for RHR

### PR DESCRIPTION
**Pull request by camhashemi**
@cvializ @andrewconner @JRuff42 

went with something simple, re-using what we have for invites. the backend-endpoints will be there once #18229 is merged.

screen shot 2015-11-09 at 3 41 14 pm

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/profile/profileService.js [hunk 1]
```diff
@@ -51,6 +51,10 @@ angular.module('kifi')
         me[key] = val;
       });
 
+      me.orgs = me.orgs.concat(me.pendingOrgs).concat(me.potentialOrgs);
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/profile/profileService.js]: shouldn't this be a backend change? it seems odd to receive this from the backend and then immediately mutate and discard parts of it.

**Comment by cvializ** [comment]
:raised_hands: looks good, can't wait to see it live : D

File: kifi-backend/modules/shoebox/angular/src/profile/profileService.js (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/profile/profileService.js]: :+1:

**Comment by andrewconner** [comment]
Looks awesome

File: kifi-backend/modules/shoebox/angular/src/profile/profileService.js (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/profile/profileService.js]: This is a backend-change on the way. I just did this to make sure it'd be safe (it is). I'll delete and have that in one PR. Thanks.

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/profile/profileService.js]: got it, good call

**Pull request closed without merging**